examples/using-wordpress/.eslintrc

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
Kill this file — the gatsby repo has its own .eslintrc file that everything should follow